### ISSUE TITLE
replace run_example_sim?

### ISSUE THREAD

**Issue by hdugan** [USGS-R/glmtools#247]
A lot of the example codes rely on `run_example_sim` to create an output.nc file. 

Since GLMr is not being maintained, and likely to shift towards running GLM through docker, I suggest we avoid running GLM in glmtools. 

Instead we could host a small output.nc file in isnt/extdata
Currently I'm using one that is ~3.5 Mb. (2 day output for ~5 months). Could even make it smaller. 

Thoughts?

**Comment by jread-usgs** [comment]
Yeah, I was kind of wondering if we wanted to discuss making a hard pivot away from GLMr and also align w/ GLM V3 changed by simply creating `glm3tools` as a separate package (w/ a lot or most copied from it). 

But perhaps that is a bad idea and would generate unnecessary work.

**Comment by hdugan** [comment]
Hmm. Hadn't considered that. Are there major changes between inputs or outputs of GLM V3 and V2? 
If not, I think we should stick with the current package. But avoid any actual running of the model. Would be easy to change all of the examples. 

We could change GLMr to docker based code. But users would still have to install docker.

**Comment by jread-usgs** [comment]
I think the glm3.nml file is different, and not sure about any changes to the .nc file. I haven't tested GLM v3 intput/output w/ current `glmtools`. I think it depends how backwards compatible `glmtools` will remain once we move forward w/ GLM3 and dropping GLMr. I'm not sure about that.
